common/src/main/java/com/tc/async/impl/StageManagerImpl.java

```diff
@@ -89,7 +90,7 @@ private void printStats() {
             stat.logDetails(logger);
           }
         } catch (Throwable th) {
-          logger.error(th);
+          logger.error("Exception :", th);
```

Comment:
The whole reason why this prefix was added was because of slf4j's lack of support for a logger api that just takes a `Throwable`. They force us to provide a message. But we never had one with TCLogging. We were relying on the exception messages for details and I just left it like that.